pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
 name = "exasol-transformers-extension"
 version = "2.3.0"
-requires-python = ">=3.10.0,<4.0"
+requires-python = ">=3.10.0,<3.15"
```

Comment:
Ok to do? This is required to resolve the torch vulnerability as triton, one of its transitive dependencies, has a stricter upper limit on the python version.

Reply:
should be fine